docs/weaviate/model-providers/google/embeddings.md

```diff
@@ -27,6 +27,13 @@ At [import time](#data-import), Weaviate generates text object embeddings and sa
 
 ![Embedding integration illustration](../_includes/integration_google_embedding.png)
 
+:::tip Which Google service should I use?
+
+- **Google AI Studio (Gemini API)**: Simpler setup, ideal for prototyping and development. Get started quickly with just an API key.
```

Comment:
Are they both available on GCP? If that's a point of difference expect that may be a big factor for many